pyseas/maps/core.py

```diff
@@ -522,21 +505,95 @@ def create_map(subplot=(1, 1, 1),
         _last_projection = projection
     _last_extent = extent
     _plot_cycler = plt.rcParams.get('pyseas.map.trackprops', styles._dark_artist_cycler)()
+    return projection, extent
+
 
+def _setup_map_axes(ax, bg_color, extent, hide_axes):
     bg_color = bg_color or plt.rcParams.get('pyseas.ocean.color', props.dark.ocean.color)
-    if not isinstance(subplot, tuple):
-        # Allow grridspec to be passed through
-        subplot = (subplot,)
-    ax = plt.subplot(*subplot, projection=projection)
     ax.background_patch.set_facecolor(bg_color)
     if extent is not None:
         ax.set_extent(extent, crs=identity)
     if hide_axes:
         ax.axes.get_xaxis().set_visible(False)
         ax.axes.get_yaxis().set_visible(False)
-    ax.spines['geo'].set_edgecolor(plt.rcParams['axes.edgecolor'])
+    # ax.spines['geo'].set_edgecolor(plt.rcParams['axes.edgecolor'])
+
+
+def create_map(subplot=(1, 1, 1), 
+               projection='global.default', 
+               extent=None,
+               bg_color=None, 
+               hide_axes=True):
+    """Draw a GFW themed map
+
+    Parameters
+    ----------
+    subplot : tuple or GridSpec, optional
+    projection : str or cartopy.crs.Projection, optional
+    extent : 4-tuple of float or None, optional
+    bg_color : str or tuple, optional
+    hide_axes : bool, optional
+        if `true`, hide x and y axes
+    
+    Returns
+    -------
+    GeoAxes
+    """
+    projection, extent = _process_map_args(projection, extent)
+
+    if not isinstance(subplot, tuple):
+        # Allow grridspec to be passed through
+        subplot = (subplot,)
+
+    ax = plt.subplot(*subplot, projection=projection)
+    _setup_map_axes(ax, bg_color, extent, hide_axes)
+
     return ax
 
+
+def create_maps(nrows=1, ncols=1, *, 
+                projection='global.default',
+                extent=None,
+                bg_color=None,
+                hide_axes=True,
+                **kwargs):
+    """Create mulitiple map similarly to plt.subplots
```

Comment:
Typo: "Create multiple maps similar to plt.subplots"

Reply:
Fixed.